instagram/src/App.js

```diff
@@ -1,27 +1,57 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
-import dummyData from './Dummy';
-import SearchBar from './components/SearchBar/searchbar';
-import PostContainer from './components/PostContainer/postcontainer';
+import dummyData from './Dummy'
+import PostPage from './components/PostContainer/PostPage';
+import Authenticate from './components/Authentication/Authenticate';
 import PropTypes from 'prop-types';
+import Login from './components/Login/Login';
 
 class App extends Component {
+  constructor(){
+    super()
+    this.state = {
+      data : [],
+      
+    }
+  }
+  componentDidMount(){
+    this.setState({
+      data: dummyData 
+    })
+  }
+  searchFilter = (event) => {
+    const search = event.target.value;
+    const data = [];
+
+    this.state.data.filter(post =>{
```

Comment:
the `filter` method expects you to return `true` or `false`; you're using it some other way